ui/screenshot.js

```diff
@@ -18,15 +18,11 @@ const scrollToBottom = async (page) => {
 const sleep = (delay) => new Promise((resolve) => setTimeout(resolve, delay));
 
 const capture = async function (folder, name, fn, opts = {}) {
-  if (!opts.namespace) {
-    opts.namespace = 'default';
-  }
-
   try {
     const path = `${__dirname}/screenshot/${folder}/fixtures/${name}.yml`;
     if (fs.existsSync(path)) {
       exec(
-        `flipt import --create-namespace --namespace=${opts.namespace} --address=${fliptAddr} ${path}`,
+        `flipt import --address=${fliptAddr} ${path}`,
```

Comment:
Does the source file that is going to be imported have the namespace in the document?

Reply:
Yah they do